CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# /****************************************************************************
+# **
+# ** Copyright (C) 2015 Intel Corporation
```

Comment:
This file is not copyright Intel, since you've written it. Please update with your copyright.

Reply:
Are you sure? Not sure if you want Microsoft copyright in the intel repo? Since this is a contribution I figured once it's in this repo it is under Intel's control and ownership.

Reply:
I sure do. You're creating it and there's no copyright assignment. Therefore, it belongs to you or (in case of work-for-hire, which is your case) the company that hired you.